ragna/assistants/_llamafile.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from ._openai import OpenaiCompliantHttpApiAssistant
+
+
+class LlamafileAssistant(OpenaiCompliantHttpApiAssistant):
+    _API_KEY_ENV_VAR = None
+    _STREAMING_METHOD = "sse"
+    _MODEL = None
+
+    @property
+    def _url(self) -> str:
+        base_url = os.environ.get("RAGNA_LLAMAFILE_BASE_URL", "http://localhost:8080")
```

Comment:
@nenb is port 8080 the default?

Reply:
This [seems](https://github.com/ggerganov/llama.cpp/blob/master/examples/server/README.md) to be the case.